Dockerfile

```diff
@@ -0,0 +1,40 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright Contributors to the Egeria project
+
+# This Dockerfile should be run from the parent directory of the egeria-connector-sas-viya directory
+# ie
+# docker -f ./Dockerfile 
+
+FROM odpi/egeria
+
+# Thes are optional tags used to add additional metadata into the docker image
+# These may be supplied by the pipeline in future - until then they will default
+
+ARG version=2.10-SNAPSHOT
+ARG VCS_REF=unknown
```

Comment:
Like with react ui, we don't use these yet. They were setup in the base where the maven process pulls them out. I'm hoping at some point we can do this in the build yaml instead as extra metadata could be useful